.circleci/config.yml

```diff
@@ -103,15 +103,15 @@ run_coverage: &run_coverage
   - run:
       name: Run Unit Tests With Coverage
       command: |
-        pytest --cov-report=xml --cov=./
+        pytest --cov-report=xml --cov=./ --timeout 45
```

Comment:
I would like to have this part of CI, so that we get a stack trace when a test times out, instead of just having a blank